CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - `WorkerConnector.HandleWorkerConnectionFailure` has been removed and `WorkerConnector.Connect` now throws exceptions for connection errors instead. [#1365](https://github.com/spatialos/gdk-for-unity/pull/1365)
 - `WorkerConnector` no longer destroys itself in `Dispose`. [#1365](https://github.com/spatialos/gdk-for-unity/pull/1365)
+- `MultiThreadedSpatialOSConnectionHandler` has been removed. [#1367](https://github.com/spatialos/gdk-for-unity/pull/1367)
```

Comment:
And the `SetThreadingMode` on the builder right?

Reply:
Yes, thats why this is mentioned in the upgrade guide. Didn't think it belonged in the changelog